### PR TITLE
Added comment about virtual methods

### PR DESCRIPTION
**Pull request by CoolDadTx**
# Title

Added comment on virtual methods

## Summary

Added comment about using HttpMessageHandler instead of overriding virtual methods when extending the class.

## Suggested Reviewers

@davidsh @karelz

### CODE REVIEW

**Comment by karelz** [comment]
cc @Priya91 @geoffkizer @wfurt

**Review by davidsh — APPROVED** [review]
LGTM

File: xml/System.Net.Http/HttpClient.xml
Comment (mairaw) [inline xml/System.Net.Http/HttpClient.xml]: LGTM. Wondering if it would be better to say "Use a constructor overload that accepts" since there's more than one though.

/cc @stevewhims

Comment (stevewhims) [reply xml/System.Net.Http/HttpClient.xml]: The "instead" works well to flow the thought from one sentence to the next, but I'd put the "instead" at the start of the 2nd sentence. Then the reader doesn't need to wait till the end of the sentence to parse that connection. So: "Instead, use the constructor overload that accepts ...".

**Comment by mairaw** [comment]
Thanks @CoolDadTx for adding this. I've made some minor changes based on the feedback given, so we can merge this PR. I'll wait for the CI build to finish before merging. The fix should go live in our next wave of updates to production in the next day or so.

**Pull request merged**
